### PR TITLE
Remove NavigationService

### PR DESCRIPTION
**Pull request by howdyitshelena**

### CODE REVIEW

File: src/components/Views/ManualGPS/ManualGPS.js [hunk 1]
```diff
@@ -761,4 +769,4 @@ class ManualGPS extends React.PureComponent<StateProps & DispatchProps, State> {
   }
 }
 
-export default ManualGPS;
+export default withNavigation(ManualGPS);
```
Comment (czengg) [inline src/components/Views/ManualGPS/ManualGPS.js]: Not sure we even need this anymore! The new upgrade of react-navigation auto includes navigation as a prop so we don't need to do withNavigation!

File: src/components/Views/MapView/Map/Map.js [hunk 2]
```diff
@@ -37,15 +37,14 @@ export type DispatchProps = {
...
-  goToCategories: () => void,
-  goToObservationDetail: () => void,
   selectObservation: (observation: Observation) => void,
   updateObservationSource: () => void,
   listStyles: () => void
 };
 
 type Props = {
-  isFocused: boolean
+  isFocused: boolean,
```
Comment (czengg) [inline src/components/Views/MapView/Map/Map.js]: Do you mind switching this out too? New react-navigation update actually includes `isFocused` as a function on the `navigation` prop. So checking for isFocused would be `navigation.isFocused()` which returns a bool. More info on it here: https://reactnavigation.org/docs/en/navigation-prop.html

File: src/components/Views/ManualGPS/ManualGPS.js (hunk 1 above)
Comment (howdyitshelena) [reply src/components/Views/ManualGPS/ManualGPS.js]: ooo fancy, okay!

File: src/components/Views/MapView/Map/Map.js (hunk 2 above)
Comment (howdyitshelena) [reply src/components/Views/MapView/Map/Map.js]: 👍 alrighty

**Pull request merged**